packages/modal/stories/Modal.stories.tsx

```diff
@@ -157,14 +203,15 @@ export const _LargeDialogModalWithFooter = () => (
           </PrimaryButton>
         }
         closeText="Dismiss"
+        {...args}
       >
         <ModalContent />
       </LargeDialogModalWithFooter>
     )}
   </ModalStoryContainer>
 );
 
-export const _FullscreenModal = () => (
+export const _FullscreenModal = args => (
```

Comment:
❓ How come some of these story names start with an underscore and some don't?

Reply:
They'll start with an underscore if they are the name of the component used. It renders in Storybook without the underscore.

Reply:
Ahh ok, good to know, thanks!